service/src/main/java/bio/terra/pipelines/stairway/PlaceholderSetStatusToSubmittedStep.java

```diff
@@ -0,0 +1,39 @@
+package bio.terra.pipelines.stairway;
+
+import bio.terra.pipelines.common.utils.JobStatusEnum;
+import bio.terra.pipelines.service.JobsService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import java.time.Instant;
+
+public class PlaceholderSetStatusToSubmittedStep implements Step {
+  private final JobsService jobsService;
+
+  /* This is a placeholder step that only sets the status in the working map;
+  it will be replaced with real steps in future PRs */
+  public PlaceholderSetStatusToSubmittedStep(JobsService jobsService) {
+    this.jobsService = jobsService;
+  }
+
+  @Override
+  public StepResult doStep(FlightContext flightContext) throws InterruptedException {
+
+    // to add later: submit the workflow to CBAS
```

Comment:
or we might decide that "timeSubmitted" means when the USER submitted the job, in which case this step wouldn't involve submitting to CBAS. in any case this will get changed a bit as we figure out how we want our Steps to work

Reply:
yea ths jobs table should just be the life of the tsps workflow/job.  so we could have a `submitted` and `modified` columns.